### DISCUSSION TITLE
Block Cart page broken

### DISCUSSION THREAD

**Discussion by PrincetonPoh** [woocommerce/woocommerce#47257] [Checkout Flow]
Cart page just showing a grey loading image. It appears to be in conflict with woopayments. If I deactivate woopayments, it works.

**Comment by ralucaStan** [comment]
@PrincetonPoh thank you for reaching out. Sorry to hear you are having these issue.  Could you share more info about your store setup? A System Status Report would be really helpful. 

The System Status Report is found in your WordPress admin under **WooCommerce > Status** Please select
"Get system report", then "Copy for support", and then paste it here.

**Comment by lbuesen** [comment]
I have the same problem:
![image](https://github.com/woocommerce/woocommerce/assets/15086506/f8feb8bc-3469-4efd-9cfa-ae3236d2f3db)


`
### WordPress Environment ###

WordPress address (URL): https://diamond-crafters.de
Site address (URL): https://diamond-crafters.de
WC Version: 8.8.3
REST API Version: ✔ 8.8.3
Action Scheduler Version: ✔ 3.7.4
Log Directory Writable: ✔
WP Version: 6.5.3
WP Multisite: –
WP Memory Limit: 512 MB
WP Debug Mode: –
WP Cron: ✔
Language: de_DE
External object cache: –

### Server Environment ###

Server Info: Apache/2.4.59 (Unix)
PHP Version: 8.1.28
PHP Post Max Size: 128 MB
PHP Time Limit: 240
PHP Max Input Vars: 4000
cURL Version: 8.6.0
OpenSSL/3.0.13

SUHOSIN Installed: –
MySQL Version: 10.6.15-MariaDB-log
Max Upload Size: 128 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 8.8.3
WC Database Prefix: ddpu_
Datenbank-Gesamtgröße: 25.49MB
Datenbank-Datengröße: 19.31MB
Datenbank-Indexgröße: 6.18MB
ddpu_woocommerce_sessions: Daten: 0.11MB + Index: 0.02MB + Engine InnoDB
ddpu_woocommerce_api_keys: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
ddpu_woocommerce_attribute_taxonomies: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
ddpu_woocommerce_downloadable_product_permissions: Daten: 0.02MB + Index: 0.06MB + Engine InnoDB
ddpu_woocommerce_order_items: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
ddpu_woocommerce_order_itemmeta: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
ddpu_woocommerce_tax_rates: Daten: 0.02MB + Index: 0.06MB + Engine InnoDB
ddpu_woocommerce_tax_rate_locations: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
ddpu_woocommerce_shipping_zones: Daten: 0.02MB + Index: 0.00MB + Engine InnoDB
ddpu_woocommerce_shipping_zone_locations: Daten: 0.02MB + Index: 0.05MB + Engine InnoDB
ddpu_woocommerce_shipping_zone_methods: Daten: 0.02MB + Index: 0.00MB + Engine InnoDB
ddpu_woocommerce_payment_tokens: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
ddpu_woocommerce_payment_tokenmeta: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
ddpu_woocommerce_log: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
ddpu_actionscheduler_actions: Daten: 2.02MB + Index: 0.94MB + Engine InnoDB
ddpu_actionscheduler_claims: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
ddpu_actionscheduler_groups: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
ddpu_actionscheduler_logs: Daten: 1.28MB + Index: 0.39MB + Engine InnoDB
ddpu_addonlibrary_addons: Daten: 0.02MB + Index: 0.00MB + Engine InnoDB
ddpu_addonlibrary_categories: Daten: 0.02MB + Index: 0.00MB + Engine InnoDB
ddpu_aioseo_cache: Daten: 0.11MB + Index: 0.03MB + Engine InnoDB
ddpu_aioseo_crawl_cleanup_blocked_args: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
ddpu_aioseo_crawl_cleanup_logs: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
ddpu_aioseo_notifications: Daten: 0.02MB + Index: 0.06MB + Engine InnoDB
ddpu_aioseo_posts: Daten: 0.28MB + Index: 0.02MB + Engine InnoDB
ddpu_commentmeta: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
ddpu_comments: Daten: 0.02MB + Index: 0.09MB + Engine InnoDB
ddpu_e_events: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
ddpu_gla_attribute_mapping_rules: Daten: 0.02MB + Index: 0.00MB + Engine InnoDB
ddpu_gla_budget_recommendations: Daten: 0.19MB + Index: 0.14MB + Engine InnoDB
ddpu_gla_merchant_issues: Daten: 0.02MB + Index: 0.00MB + Engine InnoDB
ddpu_gla_shipping_rates: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
ddpu_gla_shipping_times: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
ddpu_jetpack_sync_queue: Daten: 0.02MB + Index: 0.06MB + Engine InnoDB
ddpu_links: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
ddpu_mailpoet_automations: Daten: 0.02MB + Index: 0.00MB + Engine InnoDB
ddpu_mailpoet_automation_runs: Daten: 0.02MB + Index: 0.08MB + Engine InnoDB
ddpu_mailpoet_automation_run_logs: Daten: 0.02MB + Index: 0.06MB + Engine InnoDB
ddpu_mailpoet_automation_run_subjects: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
ddpu_mailpoet_automation_triggers: Daten: 0.02MB + Index: 0.00MB + Engine InnoDB
ddpu_mailpoet_automation_versions: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
ddpu_mailpoet_custom_fields: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
ddpu_mailpoet_dynamic_segment_filters: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
ddpu_mailpoet_feature_flags: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
ddpu_mailpoet_forms: Daten: 0.02MB + Index: 0.00MB + Engine InnoDB
ddpu_mailpoet_log: Daten: 0.02MB + Index: 0.00MB + Engine InnoDB
ddpu_mailpoet_migrations: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
ddpu_mailpoet_newsletters: Daten: 0.02MB + Index: 0.05MB + Engine InnoDB
ddpu_mailpoet_newsletter_links: Daten: 0.02MB + Index: 0.05MB + Engine InnoDB
ddpu_mailpoet_newsletter_option: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
ddpu_mailpoet_newsletter_option_fields: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
ddpu_mailpoet_newsletter_posts: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
ddpu_mailpoet_newsletter_segment: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
ddpu_mailpoet_newsletter_templates: Daten: 3.44MB + Index: 0.00MB + Engine InnoDB
ddpu_mailpoet_scheduled_tasks: Daten: 0.03MB + Index: 0.03MB + Engine InnoDB
ddpu_mailpoet_scheduled_task_subscribers: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
ddpu_mailpoet_segments: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
ddpu_mailpoet_sending_queues: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
ddpu_mailpoet_settings: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
ddpu_mailpoet_statistics_bounces: Daten: 0.02MB + Index: 0.00MB + Engine InnoDB
ddpu_mailpoet_statistics_clicks: Daten: 0.02MB + Index: 0.05MB + Engine InnoDB
ddpu_mailpoet_statistics_forms: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
ddpu_mailpoet_statistics_newsletters: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
ddpu_mailpoet_statistics_opens: Daten: 0.02MB + Index: 0.08MB + Engine InnoDB
ddpu_mailpoet_statistics_unsubscribes: Daten: 0.02MB + Index: 0.05MB + Engine InnoDB
ddpu_mailpoet_statistics_woocommerce_purchases: Daten: 0.02MB + Index: 0.08MB + Engine InnoDB
ddpu_mailpoet_stats_notifications: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
ddpu_mailpoet_subscribers: Daten: 0.02MB + Index: 0.23MB + Engine InnoDB
ddpu_mailpoet_subscriber_custom_field: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
ddpu_mailpoet_subscriber_ips: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
ddpu_mailpoet_subscriber_segment: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
ddpu_mailpoet_subscriber_tag: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
ddpu_mailpoet_tags: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
ddpu_mailpoet_user_agents: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
ddpu_mailpoet_user_flags: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
ddpu_nm_personalized: Daten: 0.02MB + Index: 0.00MB + Engine InnoDB
ddpu_novalnet_subscription_details: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
ddpu_novalnet_transaction_detail: Daten: 0.02MB + Index: 0.05MB + Engine InnoDB
ddpu_novalnet_webhook_history: Daten: 0.02MB + Index: 0.00MB + Engine InnoDB
ddpu_options: Daten: 6.08MB + Index: 0.17MB + Engine InnoDB
ddpu_postmeta: Daten: 1.45MB + Index: 0.44MB + Engine InnoDB
ddpu_posts: Daten: 1.52MB + Index: 0.06MB + Engine InnoDB
ddpu_storeabill_documentmeta: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
ddpu_storeabill_documents: Daten: 0.02MB + Index: 0.06MB + Engine InnoDB
ddpu_storeabill_document_itemmeta: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
ddpu_storeabill_document_items: Daten: 0.02MB + Index: 0.05MB + Engine InnoDB
ddpu_storeabill_document_noticemeta: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
ddpu_storeabill_document_notices: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
ddpu_storeabill_journals: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
ddpu_termmeta: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
ddpu_terms: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
ddpu_term_relationships: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
ddpu_term_taxonomy: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
ddpu_tm_taskmeta: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
ddpu_tm_tasks: Daten: 0.03MB + Index: 0.02MB + Engine InnoDB
ddpu_usermeta: Daten: 0.13MB + Index: 0.03MB + Engine InnoDB
ddpu_users: Daten: 0.02MB + Index: 0.05MB + Engine InnoDB
ddpu_wcpdf_invoice_number: Daten: 0.02MB + Index: 0.00MB + Engine InnoDB
ddpu_wcpdf_packing_slip_number: Daten: 0.02MB + Index: 0.00MB + Engine InnoDB
ddpu_wc_admin_notes: Daten: 0.06MB + Index: 0.00MB + Engine InnoDB
ddpu_wc_admin_note_actions: Daten: 0.05MB + Index: 0.02MB + Engine InnoDB
ddpu_wc_category_lookup: Daten: 0.02MB + Index: 0.00MB + Engine InnoDB
ddpu_wc_customer_lookup: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
ddpu_wc_download_log: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
ddpu_wc_orders: Daten: 0.02MB + Index: 0.11MB + Engine InnoDB
ddpu_wc_orders_meta: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
ddpu_wc_order_addresses: Daten: 0.02MB + Index: 0.06MB + Engine InnoDB
ddpu_wc_order_coupon_lookup: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
ddpu_wc_order_operational_data: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
ddpu_wc_order_product_lookup: Daten: 0.02MB + Index: 0.06MB + Engine InnoDB
ddpu_wc_order_stats: Daten: 0.02MB + Index: 0.05MB + Engine InnoDB
ddpu_wc_order_tax_lookup: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
ddpu_wc_product_attributes_lookup: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
ddpu_wc_product_download_directories: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
ddpu_wc_product_meta_lookup: Daten: 0.02MB + Index: 0.09MB + Engine InnoDB
ddpu_wc_rate_limits: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
ddpu_wc_reserved_stock: Daten: 0.02MB + Index: 0.00MB + Engine InnoDB
ddpu_wc_tax_rate_classes: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
ddpu_wc_webhooks: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
ddpu_wdr_order_discounts: Daten: 0.02MB + Index: 0.00MB + Engine InnoDB
ddpu_wdr_order_item_discounts: Daten: 0.02MB + Index: 0.05MB + Engine InnoDB
ddpu_wdr_rules: Daten: 0.02MB + Index: 0.00MB + Engine InnoDB
ddpu_woocommerce_gzd_dhl_im_products: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
ddpu_woocommerce_gzd_dhl_im_product_services: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
ddpu_woocommerce_gzd_packaging: Daten: 0.02MB + Index: 0.00MB + Engine InnoDB
ddpu_woocommerce_gzd_packagingmeta: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
ddpu_woocommerce_gzd_shipmentmeta: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
ddpu_woocommerce_gzd_shipments: Daten: 0.02MB + Index: 0.05MB + Engine InnoDB
ddpu_woocommerce_gzd_shipment_itemmeta: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
ddpu_woocommerce_gzd_shipment_items: Daten: 0.02MB + Index: 0.06MB + Engine InnoDB
ddpu_woocommerce_gzd_shipment_labelmeta: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
ddpu_woocommerce_gzd_shipment_labels: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
ddpu_woocommerce_gzd_shipping_provider: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
ddpu_woocommerce_gzd_shipping_providermeta: Daten: 0.05MB + Index: 0.03MB + Engine InnoDB
ddpu_wpforms_logs: Daten: 0.02MB + Index: 0.00MB + Engine InnoDB
ddpu_wpforms_payments: Daten: 0.02MB + Index: 0.14MB + Engine InnoDB
ddpu_wpforms_payment_meta: Daten: 0.02MB + Index: 0.05MB + Engine InnoDB
ddpu_wpforms_tasks_meta: Daten: 0.02MB + Index: 0.00MB + Engine InnoDB
ddpu_wpmm_subscribers: Daten: 0.02MB + Index: 0.00MB + Engine InnoDB
ddpu_wt_sc_coupon_lookup: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB

### Post Type Counts ###

attachment: 187
custom_css: 1
document_template: 5
elementor_library: 1
elementor-hf: 1
mailpoet_page: 1
nav_menu_item: 7
page: 16
post: 2
product: 42
product_variation: 10
revision: 25
shop_coupon: 1
shop_order_placehold: 1
wp_navigation: 1
wpcf7_contact_form: 1

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (36) ###

Advanced Product Search For WooCommerce PRO: von aThemeArt – 1.2.2
All in One SEO: von All in One SEO Team – 4.6.2
Classic Widgets: von WordPress-Mitwirkende – 0.3
Codestar Framework: von Codestar – 2.1.8
Contact Form 7: von Takayuki Miyoshi – 5.9.4
Elementor: von Elementor.com – 3.21.5
ElementsKit Lite: von Wpmet – 3.1.3
Google Listings and Ads: von WooCommerce – 2.6.9
Site Kit von Google: von Google – 1.126.0
Elementor Header & Footer Builder: von Brainstorm Force
Nikhil Chavan – 1.6.28

Limit Login Attempts Reloaded: von Limit Login Attempts Reloaded – 2.26.9
MailPoet: von MailPoet – 4.50.0
One Click Demo Import: von OCDI – 3.2.1
One Stop Shop für WooCommerce: von vendidero – 1.6.2
Pinterest für WooCommerce: von WooCommerce – 1.3.24
Premium Addons for Elementor: von Leap13 – 4.10.31
Zahlungs-Plugins für PayPal WooCommerce: von Zahlungs-Plugins
support@zahlungsplugins.com – 1.0.47

ShopEngine: von Wpmet – 4.6.4
Smart Variation Swatches for WooCommerce PRO: von aThemeArt – 1.7.6
Strato Assistant: von Strato – 7.3.0
Theme Editor: von mndpsingh287 – 2.8
TikTok: von TikTok – 1.2.5
Unlimited Elements for Elementor: von Unlimited Elements – 1.5.106
CTX Feed: von WebAppick – 6.4.29
WF Cookie Consent: von wunderfarm – 1.2.0
Germanized für WooCommerce Pro: von vendidero – 3.10.2
Germanized für WooCommerce: von vendidero – 3.16.5
WooCommerce Modal Fly Cart  + Ajax add to cart PRO: von aThemeArt – 1.2.1
WooPayments: von Automattic – 7.6.0
PDF Invoices & Packing Slips for WooCommerce: von WP Overnight – 3.8.2
WooCommerce Shipping & Tax: von WooCommerce – 2.5.6
WooCommerce: von Automattic – 8.8.3
LightStart - Maintenance Mode, Coming Soon and Landing Page Builder: von Themeisle – 2.6.11
WP-Optimize - Clean, Compress, Cache: von David Anderson
Ruhani Rabin
Team Updraft – 3.3.2

WPForms Lite: von WPForms – 1.8.8.3
Smart Coupons For WooCommerce Coupons: von WebToffee – 1.7.0

### Inactive Plugins (10) ###

Akismet Anti-spam: Spam Protection: von Automattic - Anti-spam Team – 5.3.2
Breadcrumb NavXT: von John Havlik – 7.3.0
Coupon Creator: von Brian Jessee – 3.4.0
Flexible Checkout Fields: von WP Desk – 4.1.5
Flexible Product Fields: von WP Desk – 2.4.0
GetGenie AI: von getgenieai – 3.9.4
Image Optimizer by Elementor – Compress, Resize and Optimize Images: von Elementor.com – 1.3.0
Jetpack: von Automattic – 13.4.1
Performance Lab: von WordPress Performance Team – 3.0.0
Wp Ultimate Review: von Wpmet – 2.3.2

### Dropin Plugins () ###

advanced-cache.php: advanced-cache.php

### Must Use Plugins (2) ###

WordPress automation by Installatron: von  –
WordPressSecureMode.php: von  –

### Settings ###

API Enabled: –
Force SSL: ✔
Currency: EUR (€)
Currency Position: right_space
Thousand Separator: .
Decimal Separator: ,
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: –
Enforce Approved Product Download Directories: ✔
HPOS feature screen enabled: ✔
HPOS feature enabled: ✔
Order datastore: Automattic\WooCommerce\Internal\DataStores\Orders\OrdersTableDataStore
HPOS data sync enabled: –

### Logging ###

Enabled: ✔
Handler: Automattic\WooCommerce\Internal\Admin\Logging\LogHandlerFileV2
Retention period: 30 Tage
Level threshold: –
Log directory size: 75 kB

### WC Pages ###

Shop-Basis: #7 - /shop/
Warenkorb: #8 - /warenkorb/
Kasse: #9 - /kasse/
Mein Konto: #10 - /mein-konto/
Allgemeine Geschäftsbedingungen: #170 - /agb/

### Theme ###

Name: Grocery Store PRO
Version: 1.2.7
Author URL: https://athemeart.com/
Child Theme: ❌ – Wenn du WooCommerce mit einem Theme
das du nicht selbst entwickelt hast
an eigene Bedürfnisse anpasst
dann empfehlen wir die Verwendung eines Child-Themes. Siehe Wie du ein Child-Theme erstellst (engl.)

WooCommerce Support: ✔

### Templates ###

Overrides: grocery-store-pro/woocommerce/cart/mini-cart.php
grocery-store-pro/woocommerce/cart/proceed-to-checkout-button.php
grocery-store-pro/woocommerce/content-product.php
grocery-store-pro/woocommerce/content-widget-product.php
grocery-store-pro/woocommerce/loop/add-to-cart.php
grocery-store-pro/woocommerce/single-product/add-to-cart/external.php
grocery-store-pro/woocommerce/single-product/add-to-cart/simple.php
grocery-store-pro/woocommerce/single-product/add-to-cart/variation-add-to-cart-button.php
grocery-store-pro/woocommerce/single-product/meta.php
grocery-store-pro/woocommerce/single-product/tabs/additional-information.php
grocery-store-pro/woocommerce/single-product/tabs/description.php
grocery-store-pro/woocommerce/single-product/title.php
grocery-store-pro/woocommerce/single-product-reviews.php


### WooPayments ###

Version: 7.6.0
Connected to WPCOM: Ja
WPCOM Blog ID: 229344436
Account ID: acct_1OoXXLFgZs1mGkNu
Payment Gateway: Aktiviert
Test Mode: Deaktiviert
Enabled APMs: card,giropay
WooPay: Nicht berechtigt
Apple Pay / Google Pay: Aktiviert (product,cart,checkout)
Fraud Protection Level: basic
Multi-currency: Aktiviert
Public Key Encryption: Deaktiviert
Auth and Capture: Aktiviert
Documents: Aktiviert
Logging: Deaktiviert

### Admin ###

Enabled Features: activity-panels
analytics
product-block-editor
coupons
core-profiler
customize-store
customer-effort-score-tracks
import-products-task
experimental-fashion-sample-products
shipping-smart-defaults
shipping-setting-tour
homescreen
marketing
mobile-app-banner
navigation
onboarding
onboarding-tasks
product-variation-management
product-virtual-downloadable
product-external-affiliate
product-grouped
product-linked
product-pre-publish-modal
remote-inbox-notifications
remote-free-extensions
payment-gateway-suggestions
shipping-label-banner
subscriptions
store-alerts
transient-notices
woo-mobile-welcome
wc-pay-promotion
wc-pay-welcome-page

Disabled Features: minified-js
new-product-management-experience
product-custom-fields
settings
async-product-editor-category-field
launch-your-store

Daily Cron: ✔ Next scheduled: 2024-05-13 11:02:12 +00:00
Options: ✔
Notes: 82
Onboarding: completed

### MailPoet ###

Sending Method: PHPMail
Send all site's emails with: Standard-WordPress-Versandmethode
Task Scheduler method: Action Scheduler
Cron ping URL: https://diamond-crafters.de?mailpoet_router&endpoint=cron_daemon&action=ping

### Action Scheduler ###

Abgeschlossen: 2.551
Oldest: 2024-04-12 11:28:03 +0000
Newest: 2024-05-13 10:30:00 +0000

Fehlgeschlagen: 7
Oldest: 2024-02-13 21:51:12 +0000
Newest: 2024-05-13 10:30:00 +0000

Ausstehend: 19
Oldest: 2024-05-13 10:32:00 +0000
Newest: 2024-05-19 21:29:05 +0000


### Status report information ###

Generated at: 2024-05-13 10:31:14 +00:00
`

**Comment by senadir** [comment]
Can you disable WP-Optimize and test again?

**Comment by lbuesen** [comment]
without WP-Optimize it works again - thank you :)

**Reply by senadir** [reply]
Good, I reported this to WP Optimize
https://wordpress.org/support/topic/wp-optimze-breaks-cart-checkout-pages-in-woocommerce
